rubato/structure/gameobject/tilemap/tiled.py

```diff
@@ -49,7 +49,7 @@ def __init__(
 
         self._scale = scale
         self._collider_tag = collider_tag
-        self._rects = []
+        self._rects = GameObject()
```

Comment:
still need to clone this in the clone, but I'll do that in reorder branch as I've done it already anyways.

Reply:
ok